### PR TITLE
docs(highlights): add highlights for 0.26.3

### PR DESCRIPTION
**Pull request by orhun**
🧀

### CODE REVIEW

**Review by Valentin271 — APPROVED** [review]
Looks all good!

**Comment by kdheepak** [comment]
Do we want to also announce `ratatui-macros`?

```rust
use ratatui_macros::{span, line, text};

let name = "world!";

let raw_greeting = span!("hello {name}");
let styled_greeting = span!(Modifier::BOLD; "hello {name}");

let line = line!["hello", format!("{name}")];
let line = line!["bye"; 2];

let text = text!["hello", format!("{name}")];
let text = text!["bye"; 2];
```

**Comment by kdheepak** [comment]
Or maybe we can wait for `ratatui-macros` until more tutorial material is written on the website.

**Review by kdheepak — APPROVED** [review]

File: src/content/docs/highlights/v0.26.3.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+title: v0.26.3
+---
+
+<https://github.com/ratatui-org/ratatui/releases/tag/v0.26.3>
+
+## Ratatui Forum 🌐
+
+We are happy to announce a brand new platform for Rust & TUI enthusiasts:
```
Comment (joshka) [inline src/content/docs/highlights/v0.26.3.md:9]: Perhaps change platform to forum?

File: src/content/docs/highlights/v0.26.3.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+[Ratatui Forum](https://forum.ratatui.rs) 🐭
```
Comment (joshka) [inline src/content/docs/highlights/v0.26.3.md]:
```suggestion
We are happy to announce a brand new
[Ratatui Forum](https://forum.ratatui.rs) 🐭 for Rust & TUI enthusiasts.
```

**Review by joshka — CHANGES_REQUESTED** [review]

**Review by joshka — APPROVED** [review]

File: src/content/docs/highlights/v0.26.3.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+[`assert_buffer_eq`](https://docs.rs/ratatui/latest/ratatui/macro.assert_buffer_eq.html) is now
+deprecated in favor of `assert_eq`:
+
+```diff
+-assert_buffer_eq!(actual, expected);
++assert_eq!(actual, expected);
+```
+
+We also introduced a new macro called `assert_buffer_lines` for checking if `TestBackend`'s buffer
+is equal to the expected lines.
```
Comment (EdJoPaTo) [inline src/content/docs/highlights/v0.26.3.md:109]: It's a new function on `TestBackend`. `TestBackend::assert_buffer_lines`. It's to simplify `TestBackend::assert_buffer(&Buffer::with_lines(…))`. The current wording sounds like it's a replacement for any `assert_eq!()` with buffers.

Comment (joshka) [reply src/content/docs/highlights/v0.26.3.md:109]: Feel like slapping a quick PR together for this so this comment doesn't get lost?

Comment (EdJoPaTo) [reply src/content/docs/highlights/v0.26.3.md:109]: https://github.com/ratatui-org/ratatui-website/pull/612

**Pull request merged**